### PR TITLE
Feature selection and documentation

### PR DESCRIPTION
**Pull request by chantelleleveille**
# Feature selection

Features used in development were pruned and the final set of updated features remain. 
- all calls to the old density column have been removed
- features not used in analysis workflows have been dropped
- features for TFE have been updated

# Feature documentation
- documentation tables have been updated
- glossary definitions for TFE have been updated. 

TFE outputs are being generated here: `/allen/aics/assay-dev/users/Chantelle/colorizer/final/`

### CODE REVIEW

**Comment by jcass11** [comment]
@chantelleleveille I will plan to take on fixing the test errors this weekend if I find time or first thing monday

File: nuc_morph_analysis/lib/preprocessing/add_features.py [hunk 1]
```diff
@@ -570,17 +574,18 @@ def sum_mitotic_events_along_full_track(df0, feature_list=[]):
...
-        'has_mitotic_neighbor_formation',
-        'has_mitotic_neighbor_breakdown_forward_dilated',
-        'has_mitotic_neighbor_formation_backward_dilated',
         'has_mitotic_neighbor',
-        'has_mitotic_neighbor_dilated',
         'has_dying_neighbor',
-        'has_dying_neighbor_forward_dilated',
-        'number_of_frame_of_death_neighbors'
+        
+        # 'number_of_frame_of_breakdown_neighbors',
```
Comment (jcass11) [inline nuc_morph_analysis/lib/preprocessing/add_features.py:580]: Should these commented out list items just be removed?

**Comment by jcass11** [comment]
Tests are fixed. Re-running `run_all_manuscript_workflows.py` for good measure.

**Review by jcass11 — APPROVED** [review]
====== Successes ======
figure_1_dataset (3050.4s) (56.2GB)
figure_3_s4_height_density (3218.3s) (50.0GB)
figure_4_s8_volume_trajectories (726.0s) (43.8GB)
figure_5_s9_s10_local_growth (2187.3s) (53.8GB)
figure_6_s11_compensation (730.5s) (43.9GB)
figure_7_s13_s14_lineage (899.5s) (43.8GB)
figure_s12_feeding_control (561.2s) (26.8GB)
figure_s15_linear_regression_model (1950.7s) (43.9GB)
figure_s19_transition_point (678.2s) (43.8GB)
figure_s1_cell_health (670.0s) (43.8GB)
figure_s20_precision_error (6.5s) 
figure_s2_s3_s18_segmentation_model_validation (132.0s) (4.4GB)
figure_s6_s16_inhibitors (606.6s) (14.7GB)
figure_s7_growth_outliers (733.8s) (44.1GB)
====== 14 succeeded  ======
YES!

File: nuc_morph_analysis/lib/visualization/label_tables.py [hunk 2]
```diff
@@ -78,15 +75,8 @@ def get_scale_factor_table(dataset="all_baseline"):
...
-            ): (pix_size/2.5)**2, # resolution_level 1 is 2.5x downsampled
-        ("2d_area_pseudo_cell"): (pix_size/2.5)**2, # resolution_level 1 is 2.5x downsampled
-        ("inv_cyto_density"): 1 / (pix_size/2.5)**2, # resolution level =1 is 2.5x downsampled
-         (
-            "2d_intensity_min_edge",
-            "2d_intensity_mean_edge",
-            "2d_intensity_max_edge",
-        ): pix_size/2.5, # resolution level =1 is 2.5x downsampled
+            ): (pix_size/2.5)**2,
+        ("2d_area_pseudo_cell"): (pix_size/2.5)**2,
```
Comment (cfrick13) [inline nuc_morph_analysis/lib/visualization/label_tables.py]: nit: can delete this line. its  duplicating line 77 above.

**Review by cfrick13 — APPROVED** [review]
Excellent work! So exciting to see this pushed to the finish line!

File: nuc_morph_analysis/lib/visualization/label_tables.py (hunk 2 above)
Comment (chantelleleveille) [reply nuc_morph_analysis/lib/visualization/label_tables.py]: done!

**Pull request merged**